src/interactions/test/conftest.py

```
@@ -1,34 +1,23 @@
 import pytest
-from django.contrib.auth import get_user_model
-
-from about_us.models import AboutUs
-from news.models import NewsPage, Comment
-
-from interactions.models import PageReaction, ReactionType, CommentReaction
 
 
-@pytest.fixture
-def user():
-    return get_user_model().objects.create(username="test_user")
+from about_us.models import AboutUs
+from news.models import NewsHome
 
 
-@pytest.fixture
-def user2():
-    return get_user_model().objects.create(username="test_user2")
+from peoplefinder.test.factories import UserWithPersonFactory
+from news.factories import NewsPageFactory
 
 
 @pytest.fixture
-def user3():
-    return get_user_model().objects.create(username="test_user3")
+def user():
+    return UserWithPersonFactory()
 
 
 @pytest.fixture
 def news_page():
-    return NewsPage.objects.create(
-        title="News",
-        depth=1,
-        path="page2",
-    )
+    news_home_page = NewsHome.objects.first()
```

`first` returns `NewsHome | None`
This means that there is a chance you create a `NewsPage` with no parent. Probably worth changing this to `get` so that it fails hard if there is no `NewsHome`